Classes/Command/ExtensionCommandController.php

```diff
@@ -161,6 +162,16 @@ public function dumpAutoloadCommand()
     }
 
     /**
+     * List all installed extensions
+     *
+     * @return void
+     */
+    public function listInstalledCommand()
+    {
+        $this->outputLine('%s', [implode(',', ExtensionManagementUtility::getLoadedExtensionListArray())]);
```

Comment:
Please also make the output a bit more human readable. if you also need machine readable output, please add a switch for that (e.g. `--raw`)